website/blog/2020-07-30-7.11.0.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title:  "7.11.0 Released: ECMAScript 2021 support in preset-env, TypeScript 4.0 support, printing config and the future of `babel-eslint`"
+author: Huáng Jùnliàng
+authorURL: https://twitter.com/JLHwung
+date:   2020-07-30 0:00:00
+categories: announcements
+share_text: "Babel 7.11.0 Released"
+---
+
+
+
+We just released a new minor version of Babel!
+
+This 7.11 release includes:
+
+- `preset-env` supports Logical Assignment and Numeric Separator and Export Namespace
+- TypeScript 4.0 Support
+- Parser support for stage-1 decimal `7.11m`
+- Print effective Babel config on an input source `BABEL_SHOW_CONFIG_FOR`
+- The future of `babel-eslint`: `@babel/eslint-parser`
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.11.0).
+
+<!-- truncate -->
+
+Also if you have any questions, we've enabled [GitHub Discussions](https://github.com/babel/babel/discussions) on our repository!
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## ECMAScript 2021 support ([#11864](https://github.com/babel/babel/pull/11864))
+
+During the last [meeting](https://github.com/tc39/agendas/blob/master/2020/09.md), TC39 moved both the logical assignment and numeric separator proposals to Stage 4! Babel have supported these proposals via `@babel/plugin-proposal-logical-assignment-operators` and `@babel/plugin-proposal-numeric-separators`, now they are included in `@babel/preset-env` and compiled based on your targets, just like any other ECMAScript features.
+
+### [Logical Assignment](https://babeljs.io/docs/en/babel-plugin-proposal-logical-assignment-operators)
+Logical assignment offers a shorthand notation combining logical operators and assignment expression:
+
+```javascript
+this.disabled ??= false;
+this.disabled = this.disabled ?? (this.disabled = false);
+
+clicked &&= !isDoubleClicked();
+clicked && (clicked = !isDoubleClicked());
+
+hasDups ||= (prev === cur);
+hasDups || (hasDups = (prev === cur));
```

Comment:
Maybe here we could show more real world examples instead of the exact compilation?

```suggestion
clicked &&= !isDoubleClicked();
clicked = clicked && !isDoubleClicked();

hasDups ||= (prev === cur);
if (!hasDup) hasDups = (prev === cur);
```